From e4823210911c344e668eba7783f170816777f388 Mon Sep 17 00:00:00 2001
From: Pavel Tikhomirov <ptikhomirov@virtuozzo.com>
Date: Fri, 14 Jan 2022 12:12:12 +0300
Subject: [PATCH 216/249] ci: run tests for old mount engine

Now when we switched to mount-v2 by default to check old mount engine we
need to explicitly run with --mntns-compat-mode option.

Note that if the feature move_mount_set_group is not supported then
regular run will just fallback to old mount engine and then we don't
need separate run with --mntns-compat-mode.

Signed-off-by: Pavel Tikhomirov <ptikhomirov@virtuozzo.com>
---
 scripts/ci/run-ci-tests.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/ci/run-ci-tests.sh b/scripts/ci/run-ci-tests.sh
index 5ef3965db..dd7c49d63 100755
--- a/scripts/ci/run-ci-tests.sh
+++ b/scripts/ci/run-ci-tests.sh
@@ -201,6 +201,10 @@ fi
 
 # shellcheck disable=SC2086
 ./test/zdtm.py run -a -p 2 --keep-going $ZDTM_OPTS
+if criu/criu check --feature move_mount_set_group; then
+	# shellcheck disable=SC2086
+	./test/zdtm.py run -a -p 2 --mntns-compat-mode --keep-going $ZDTM_OPTS
+fi
 
 # shellcheck disable=SC2086
 ./test/zdtm.py run -a -p 2 --keep-going --criu-config $ZDTM_OPTS
-- 
2.35.1

